packages/yuudachi/locales/en-US/translation.json

```diff
@@ -283,6 +284,23 @@
 			},
 			"duration": {
 				"success": "Successfully set duration for case {{- case}}"
+			},
+			"clear": {
+				"errors": {
+					"no_message": "Could not find message {{message_id}} in channel {{- channel}}",
+					"no_results": "Could not find messages in the provided range that are younger than 12 hours."
+				},
+				"pending": "You are about to clear {{count}} message by $t(command.mod.clear.authors, {\"count\": {{author_count}}}) sent in {{time}}. Are you sure?",
+				"pending_plural": "You are about to clear {{count}} messages by $t(command.mod.clear.authors, {\"count\": {{author_count}}}) sent in {{time}}. Are you sure?",
+				"success": "Successfully cleared {{count}} messages by $t(command.mod.clear.authors, {\"count\": {{author_count}}}) sent in {{time}}",
+				"success_plural": "Successfully cleared {{count}} messages by $t(command.mod.clear.authors, {\"count\": {{author_count}}}) sent in {{time}}.",
+				"authors": "{{count}} author",
+				"authors_plural": "{{count}} authors",
+				"buttons": {
+					"execute": "Clear {{count}} message",
+					"execute_plural": "Clear {{count}} messages"
```

Comment:
This works a bit different in i18next: https://www.i18next.com/translation-function/plurals#singular-plural